R/corPFSOS.R

```diff
@@ -0,0 +1,123 @@
+# E(PFS) helpers:
```

Comment:
please add documentation for these as a next step

Reply:
done